apps/earn-protocol/components/molecules/SidebarElements/OrderInfoDeposit.tsx

```diff
@@ -39,27 +39,55 @@ export const OrderInfoDeposit = ({
         <SimpleGrid columns={2} gap={2}>
           <Text variant="p3semi">Deposit Amount</Text>
           <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
-            {fromAmount.toString()}
+            {formatCryptoBalance(fromAmount.amount)} {fromAmount.token.symbol}
           </Text>
           {toAmount && priceImpact && (
             <>
               <Text variant="p3semi">Swap</Text>
-              <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
-                {fromAmount.toString()} {'->'} {toAmount.toString()}
-              </Text>
+              <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'flex-end' }}>
+                <Icon
+                  tokenName={fromAmount.token.symbol.toUpperCase() as TokenSymbolsList}
+                  size={20}
+                />
+                <Text
+                  variant="p3semi"
+                  className={orderInfoDepositWithdrawStyles.depositDetailsValue}
+                >
+                  {formatCryptoBalance(fromAmount.amount)} {'-> '}
+                </Text>
+                <Icon
+                  tokenName={toAmount.token.symbol.toUpperCase() as TokenSymbolsList}
+                  size={18}
+                />
+                <Text
+                  variant="p3semi"
+                  className={orderInfoDepositWithdrawStyles.depositDetailsValue}
+                >
+                  {formatCryptoBalance(toAmount.amount)}
+                </Text>
+              </div>
               <Text variant="p3semi">Price Impact</Text>
               <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
-                {priceImpact.price.toString()} ({priceImpact.impact.toString()})
+                {formatCryptoBalance(priceImpact.price.value)}{' '}
+                {(priceImpact.price.quote as IToken).symbol} (
+                {formatPercent(priceImpact.impact.value, {
+                  precision: 2,
+                })}
+                )
               </Text>
               <Text variant="p3semi">Slippage</Text>
               <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
-                {slippage.toString()}
+                {formatPercent(slippage.value, { precision: 2 })}
               </Text>
             </>
           )}
           <Text variant="p3semi">Transaction Fee</Text>
           <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
-            {transactionFeeLoading ? 'Loading...' : transactionFee}
+            {transactionFeeLoading
+              ? 'Loading...'
```

Comment:
I think we can use here SkeletonLine

`<SkeletonLine style={{ display: 'inline-block' }} width={200} height={30} />` (props to be configured)